### PR TITLE
Gamma into main

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
I think this is correct but someone PTAL @sallyom @lance

### CODE REVIEW

**Review by sallyom — APPROVED** [review]
this lgtm!

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -2,11 +2,12 @@ name: Lint Charts
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main,release-1.0.gamma ]
```
Comment (lance) [inline .github/workflows/test.yaml:5]: I would remove these updates to the gamma branch

File: .tekton/client-server-1-0-gamma-pull-request.yaml
Comment (lance) [inline .tekton/client-server-1-0-gamma-pull-request.yaml:1]: This should be named `.tekton/client-server-pull-request.yaml`, and there will need to be changes to `metadata` and `spec` here and in the push pipeline. But I see also that there are these two files below named correctly. They have some changes such as https://github.com/securesign/sigstore-ocp/pull/192/files#diff-b5efe344fc973beeaf5282a88fea4c85e30dc8c113a6c18629c5e7bee17af509L146-L151 which I'm not really sure about.

**Review by lance — COMMENTED** [review]
Mostly this seems OK with a few small changes. But I wonder if it isn't just more straightforward and possibly safer to force push `release-1-0-gamma` to main. E.g. `git push -f origin release-1-0-gamma:main`.

**Pull request closed without merging**